examples/story-book/server/api/story/index.post.ts

```diff
@@ -0,0 +1,41 @@
+import { exec } from 'child_process'
+import type { ChildProcess } from 'child_process'
+
+type Request = {
+    prompt: string;
+    pages: number;
+}
+
+export const commands: Record<string, ChildProcess>= {}
+
+export default defineEventHandler(async (event) => {
+    const request = await readBody(event) as Request
+
+    if (!request.prompt) {
+        throw createError({
+            statusCode: 400,
+            statusMessage: 'prompt is required'
+        });
+    }
+
+    if (!request.pages) {
+        throw createError({
+            statusCode: 400,
+            statusMessage: 'pages are required'
+        });
+    }
+
+    const command = `gptscript story-book.gpt --story ${request.prompt}, '--pages', ${`${request.pages}`}`
```

Comment:
We do have a node wrapper: https://github.com/gptscript-ai/node-gptscript

Would this be a good thing to use here?

Reply:
Oh yeah, definitely. That will be way better.

Reply:
Done! I also added some additional features that improve the UX of waiting for a story.